tools/appveyor.psm1

```diff
@@ -108,6 +108,20 @@ function Invoke-AppVeyorInstall
         Update-AppveyorBuild -message $buildName
     }
 
+    # Generate credentials for appveyor remoting tests.
```

Comment:
AppVeyor remotely does the same thing so it presents the same issue
https://github.com/PowerShell/Remotely/blob/master/appveyor.yml

However, I don't feel this is a conflict since credential creation is done each time before remoting tests or remotely tests are run.  Enable-rdp.ps1 can still be run as needed without affecting if/when the tests are run again.